compiler/codegen/library.cpp

```diff
@@ -261,6 +268,128 @@ static void printMakefileLibraries(fileinfo makefile, std::string name) {
   fprintf(makefile.fptr, " %s %s\n\n", libraries.c_str(), libname.c_str());
 }
 
+// Helper to convert command lines flags from make syntax (using parens) to
+// CMake syntax (using brackets)
```

Comment:
Is this about `$(BLA)` vs `${BLA}` ? If so can you update the comment to use that example? Also can you make the code check that there is a `$` before the `(` being replaced? It could assert that, for now, with `INT_ASSERT`.

Reply:
It is, and for sure. For the `$` check, do you mean something like `INT_ASSERT(str[pos-1] == '$')` before switching to a bracket?